packages/contract/lib/execute.js

```diff
@@ -140,7 +149,10 @@ const execute = {
       }
 
       execute
-        .prepareCall(constructor, methodABI, args)
+        .prepareCall(constructor, methodABI, args, {
+          id: constructor.network_id,
+          blockLimit: 0
```

Comment:
Is `0` really a sensible default for this?

Reply:
@eggplantzzz Yes you are correct, it doesn't matter during calls types calls.

Reply:
@eggplantzzz do you have a suggestion how should it look like?

Reply:
@eggplantzzz Please review it again, I removed blockLimit parameter from **_detectNetworkArguments** which is not required during calls.

Reply:
@gnidan @eggplantzzz Please have a look.

Reply:
ping